ZCS-1417

mihaipopahtp opened the pull request:
- adding Vietnamese translation

silentsakky commented on WebRoot/js/ajax/config/msgs/AjxMsg_vi.properties:
```diff
@@ -0,0 +1,157 @@
+# 
```
I think location of this file looks wrong

mihaipopahtp replied:
I removed the vi file since no other localized file was there. Only en-US.
I tried to mimic the en-US file structure and copied vi as well.

mihaipopahtp replied:
I have the same in zm-admin-ajax\WebRoot\js\ajax\config\msgs\. Only en-US file so removing _vi from there as well.

silentsakky replied:
thanks, probably I will look at removing english version file as well if not required

silentsakky committed:
ZCS-1417
- adding Vietnamese translation
- removing file with wrong location
- adding ZCS-2592 updates to smart quotes
- changes after linguistic testing
- updating capitalization for month names (20171222)
